Multisig/UI/Settings/AppSettingsViewController/FeatureToggleTableViewController.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  FeatureToggleTableViewController.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 18.01.23.
+//  Copyright © 2023 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class FeatureToggleTableViewController: UITableViewController {
+
+    enum RowID {
+        case securityCenter
+        case halt
```

Comment:
Why do we need this case ?

Reply:
I thought that I'd need to restart the app after feature toggle is changed. However, it seems to work without it.